source/help/getting-started/managing-members.md

```diff
@@ -48,30 +48,24 @@ The System Admin is typically a member of the IT staff and has all the privilege
 
 #### Inactive 
 
-A System Administrator has the ability to make a user account Inactive by going to **System Console > Teams**, and selecting a team to manage. From there they can go to the **User** page for the team, and manage a user's role. 
-
-When "Inactive" is selected, the user will no longer be able to log into the system. 
+A System Administrator can turn a user account Inactive by going to **System Console > Teams**, selecting a team to manage, the clicking on **User** and selecing "Make Inactive". Inactive users are removed from all teams and private groups and logged out from all devices and can no longer log in to the system. Users can be re-activated using the "Make Active" menu option. 
```

Comment:
"the" should be "then", "selecing" should be "selecting"

This is also incorrect? Inactive users aren't really removed from teams. If you turn them active again, they still belong to the team.